### PR TITLE
Remove reference to Lubuntu Software Centre

### PR DESCRIPTION
**Pull request by Caedorus**
Lubuntu Software Centre is depreciated, therefore no longer in the repos and cannot be installed. Changed it to point to gnome-software instead.

### CODE REVIEW

**Comment by Caedorus** [comment]
Fixes #588

File: userguide/dailyuse/libertine.rst [hunk 1]
```diff
@@ -72,7 +72,7 @@ From the apps list in the *Manage Libertine Containers* menu you can tap the + s
 
 Enter the package name if you know it.
 Otherwise you can search the archives for a package.
-You can make this process easier by making the *Lubuntu Software Center* your first package install.
+You can make this process easier by making a software center like gnome-software your first package install.
```
Comment (doniks) [inline userguide/dailyuse/libertine.rst:75]: I'm wondering ... did you try to install gnome-software? is it possible? does it run? does it run well enough to justify the claim that this makes the "process easier"?

It's been a long long time that I looked at this, but I think gnome-software is a rather .... whatstheword .... advanced ... sophisticated ... modern application. I'm wondering how well it actually runs in libertine. did you test it? can you comment?

Comment (Caedorus) [reply userguide/dailyuse/libertine.rst:75]: I did test it, it worked fine though I would recommend attaching a mouse for ease of use (I can't get the scaling to work yet)

Comment (Caedorus) [reply userguide/dailyuse/libertine.rst:75]: Alternatively feel free to suggest another one, the one to change it *to* isn't really the point of this PR

Comment (doniks) [reply userguide/dailyuse/libertine.rst:75]: > I did test it, (fp4 focal ota-4) it works fine though I would recommend attaching a mouse for ease of use (I can't get the scaling to work yet)

great! thanks

**Review by doniks — APPROVED** [review]

**Pull request merged**